commons/src/main/java/io/hyscale/commons/logger/TableFields.java

```diff
@@ -18,7 +18,7 @@
 public enum TableFields {
 
 	SERVICE("Service Name", 30), STATUS("Status"), AGE("Age"), REASON("Reason"), MESSAGE("Message"),
-	SERVICE_ADDRESS("Service Address", 40);
+	SERVICE_ADDRESS("Service Address", 40), REPLICA_NAME("Replica name", 40), INDEX("Index", 10);
```

Comment:
Do you think we can reduce it lower value , 10 is large number for index ??

Reply:
Sure